ibis/backends/tests/test_generic.py

```diff
@@ -34,7 +34,15 @@ def test_fillna_nullif(backend, con, expr, expected):
 
 
 @pytest.mark.notimpl(
-    ["clickhouse", "datafusion", "impala", "mysql", "postgres", "sqlite"]
+    [
+        "clickhouse",
+        "datafusion",
+        "duckdb",
```

Comment:
This just isn't possible in duckdb right now since you can't interact with non-finite values yet. This is slated to change in an upcoming release.

Should it be a `notyet`?